department-of-veterans-affairs/abd-vro
# Further code cleanup - Remove BipConditions and MockBipCeApiService.

Further code cleanup to remove unused code and simplify the logic.

## What was the problem?
As we have mock BIP API services in place, MockBipCeApiService as well as BipConditions are no longer needed. This PR removes those unused code and simplify the code logic.

Associated tickets or Slack threads:
MCP-2347

## How does this fix it?[^1]
Remove the code no longer needed.

## How to test this PR
- Step 1. Build and launch the application.
- Step 2. Test automatedClaim to see whether it is still working.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
